wiki/libraries/loom/markdown/access_widener.md

```diff
@@ -0,0 +1,79 @@
+# Access Widener
+
+Access Wideners are a system that allows developers to change the access modifiers of methods, fields, and classes from
+Minecraft.
+
+## Header
+
+Every access widener must have a header which specifies the version of the format, and the target namespace.
+
+```
+accessWidener v1 named
```

Comment:
```suggestion
accessWidener v2 named
```

I'd rather have the up-to-date stuff first to avoid anyone trying to take the first thing they see and paste bad stuff.

Reply:
Done